webnn/resources/test_data/l2_pool2d.json

```diff
@@ -0,0 +1,1509 @@
+{
+  "tests": [
+    {
+      "name": "l2Pool2d float32 4D constant tensor all positive default options",
```

Comment:
> all positive

Since each element is squared anyway, it doesn't really matter for this case whether there's a mix of positive and negative value. I'm not suggesting you change anything, just noting.

Reply:
>  it doesn't really matter for this case whether there's a mix of positive and negative value

I remembered that you once share me the operation list of avoiding catastrophic cancellation which included this l2Pool2d .